### DISCUSSION TITLE
Directions for Use?

### DISCUSSION THREAD

**Discussion by Lawlesslisa** [SobranDM/foundryvtt-wwn#85] [Q&A] [answered]
I am trying this WWN ruleset for the first time.  I do not know how to make a character with a class or how to use a weapon to roll an attack.  Is there a readme or guide somewhere explaining how to use this ruleset?  I tried to search for a video or guide on how to use this or the OSE set but was not having any luck.

Thank you for your time,
Lisa

**Comment by SobranDM** [comment]
Is there anything in particular you're having trouble with? There's no automated character generation system, so you have to add the things you know you need from the item compendiums.

If you have Discord, you can usually get a faster response from me over on the Worlds Without Number channel: https://discord.com/invite/Vd7ejK3n

**Comment by Lawlesslisa** [comment]
Hello, thanks you for responding.  I don't see a class or background compendium and don't know how to add those to my character.  Also, I do not drag and drop skills?  I have to edit each one in?  I just wasn't sure how to do  these things and didn't know if there was a guide on how to fill out the character sheet for the ruleset.

Thanks

**Comment by SobranDM** [comment]
Yeah, that's what I mean by there being little to no automation for
character creation. A class just grants you a certain hit bonus
progression, hit dice, and class features. You have to manually enter the
first two and the various features can be dragged and dropped from the item
compendium. The class field is likewise just a text field.

Attribute rolls have some automation, but not the free 14 you're granted.
You have to change that yourself.

Likewise, backgrounds are not implemented because it would require a fair
amount of automation for the user to select which tables they want to roll
on and so on. It's certainly doable, but the more automation I do, the more
difficult homebrew becomes. So you're expected to make those rolls and
update the skills accordingly.

Re: skills, when you create a new character there should be a flashing
button that says add skills, which adds all the default WWN skills. If
you're not seeing that, something has gone wrong.

I'll see about adding a short video showing the process.

**Comment by SobranDM** [comment, marked as answer]
Hopefully this helps. If you watch closely, you'll see me cut out a flub in the middle. It's not narrated or anything but should show you all the ins and outs of character creation, at least. As mentioned above, I'd encourage you to join the Discord if you have any further questions, as you can hear from me in a timely fashion.

https://youtu.be/BcgvQZodOrc

**Comment by Lawlesslisa** [comment]
Thanks for doing this video, Sobran.  It is much appreciated and helpful 👍
